test/stimulus_test.rb

```diff
@@ -4,7 +4,7 @@ class StimulusTest < ActionView::TestCase
   include Stimulus::ImportmapHelper
 
   test "import map helper with files in directories" do
-    assert_equal <<~JSON.strip, importmap_list_from("app/assets/javascripts/controllers", "app/assets/javascripts/libraries")
+    assert_json_equal <<~JSON.strip, importmap_list_from("app/assets/javascripts/controllers", "app/assets/javascripts/libraries")
```

Comment:
I believe that CI's filesystem and my local development filesystem was walking the file tree in a different order. Since I _don't think_ the importmap keys' order is important, I've changed this test to parse them as JSON and compare the two resulting objects.

Reply:
@dhh the suite is now running in full, and passing.